### PR TITLE
added packages/minicli/minicli.3.0.0

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
CI tests are all green

**Comment by UnixJunkie** [comment]
apparently, conf-m4 fails to install on alpine linux
Cf. https://ci.ocamllabs.io/log/saved/docker-build-5bd8fbce9a4de9c28bff74bba795b681/f8aba9026cbbdc8cc5e14fa31906c0ee22b95ccf

File: packages/minicli/minicli.3.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+bug-reports: "https://github.com/UnixJunkie/minicli/issues"
+dev-repo: "https://github.com/UnixJunkie/minicli.git"
+build: [
+  ["obuild" "configure"]
+  ["obuild" "build" "lib-minicli"]
+]
+install: ["obuild" "install"]
+remove: ["ocamlfind" "remove" "minicli"]
+depends: [
+  "ocamlfind"
```
Comment (hannesm) [inline packages/minicli/minicli.3.0.0/opam]: this is likely a `{build}`-only dependency, or am I wrong here?

Comment (UnixJunkie) [reply packages/minicli/minicli.3.0.0/opam]: yes, corrected

**Comment by hannesm** [comment]
thanks!

**Pull request merged**